app/views/shared/check_answers/_substantive_limitations.html.erb

```diff
@@ -12,6 +12,6 @@
   <%= check_answer_no_link(
         name: :allowable_work,
         question: t('.work'),
-        answer: @legal_aid_application.application_proceeding_types.first.substantive_scope_limitation.description
+        answer: @legal_aid_application.lead_application_proceeding_type.substantive_scope_limitation.description
```

Comment:
I think this is probably fine - it seems slightly arbitrary to just use the first proceeding anyway, using the lead proceeding certainly doesn't seem to be a worse approach.

Reply:
What SHOULD we be displaying in this partial - sounds like the scope limitations for the lead proceeding is right, as opposed to any old proceeding.  The `.first` probably dates back from when we only had one application proceeding type per application.

Reply:
It was added as part of PR #2381 so it possibly pre-dates the implementation of _lead_ _proceeding_types?